content/02.introduction.md

```diff
@@ -1,6 +1,19 @@
-#Introduction
-Set introduction for paper here
-Talk about problem, goal, and significance of paper
+# Introduction
+
+Populating knowledge bases is a daunting and arduous task that takes a significant amount of manual effort to accomplish.
+Typically, this task involves  manual curation, which is the process of curators reading and parsing published papers to extract relationships of interest. 
+This framework becomes infeasible as the rate of publications continues to increase exponentially [@doi:10.1002/asi.23329].
+A common solution towards this problem is to perform relationship extraction.
+Relationship extraction is the process of creating a machine learning system to automatically scan and extract relationships from textual sources.
```

Comment:
Has relationship extraction been proposed as a solution to this problem? It seems like someone must have proposed this in the literature in the past. If so, this should be re-worked to include that reference here.